.github/workflows/build.yml

```diff
@@ -427,6 +427,57 @@ jobs:
             library-x86_64-linux-android
           failOnError: false
 
+  alpha-release:
+    name: Alpha Release
+    if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+    permissions:
+      id-token: write
+    runs-on: ubuntu-latest
+    env:
+      NPM_TAG: 'alpha'
+    needs: 
+      - create-ios-xcframework
+      - create-android-library
+    defaults:
+      run:
+        working-directory: ./wrappers/javascript
+    timeout-minutes: 7
+    steps:
```

Comment:
A few spaces won't change much but make make it easier for the eye to follow. Between steps?